Source/Private/Merge-CodeAndLintRemarks.ps1

```diff
@@ -11,6 +11,8 @@ function Merge-CodeAndLintRemarks {
     )
     [String[]] $result = @()
 
+    # TODO Sorth the lint remarks to ensure consistent ordering.
```

Comment:
```suggestion
    # TODO Sort the lint remarks to ensure consistent ordering.
```

Reply:
Hehe, it's actually done, so I removed the TODO.